### PR TITLE
Fix problem removing properties in python 3

### LINKED ISSUE #10: Error removing keys with python 3
### Prerequisites

* [ x] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filled: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aidealista
    * Checked that there is not already provided the described functionality

### Description

Unexpected result while removing keys in python 3

### Versions

1.0.1

### PR DESCRIPTION
**Pull request by jmonterrubio**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Benefits

Molecule 2
Fix #10 

### Applicable Issues

#10

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ansible==2.5.3.0
```
Comment (dortegau) [inline test-requirements.txt]: maybe it's better to define an ansible >= 2.5.5.0

File: README.md [hunk 2]
```diff
@@ -23,14 +23,12 @@ These instructions will get you a copy of the role for your Ansible Playbook. On
 
 ### Prerequisities
 
-Ansible 2.4.3.0 version installed.
-Molecule 1.25 version installed.
-Ansible-Lint 3.4.20 version installed.
-python-consul 1.1.0 version installed
+Ansible >= 2.4.3.0
```
Comment (dortegau) [inline README.md]: is not consistent with version defined in test-requirements.txt file

File: README.md [hunk 3]
```diff
@@ -66,13 +64,8 @@ Look to the [defaults](defaults/main.yml) properties file to see the possible co
...
 
 ```sh
-molecule test
-```
-
-or
-
-```sh
-./tests/scripts/run-test.sh
+$ pipenv install -r test-requirements.txt --python 2.7
```
Comment (dortegau) [inline README.md]: It would be nice if specifies that Python 3.x could be used 🙂

File: README.md (hunk 2 above)
Comment (jmonterrubio) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (jmonterrubio) [reply README.md]: Updated

File: test-requirements.txt (hunk 1 above)
Comment (jmonterrubio) [reply test-requirements.txt]: Updated

File: README.md [hunk 4]
```diff
@@ -23,14 +23,14 @@ These instructions will get you a copy of the role for your Ansible Playbook. On
...
-python-consul 1.1.0 version installed
+Ansible 2.5.5.0
+Python 3.6 or Python 2.7
+Other combinations may work but they're not tested.
 
 Inventory destination should be a Debian environment.
 
-For testing purposes, [Molecule](https://molecule.readthedocs.io/) with [Vagrant](https://www.vagrantup.com/) as driver (with [hostmanager](https://github.com/devopsgroup-io/vagrant-hostmanager) plugin) and [VirtualBox](https://www.virtualbox.org/) as provider.
+For testing purposes, [Molecule](https://molecule.readthedocs.io/) with [Docker](https://www.docker.com/) as driver.
+See [test-requirements](test-requirements.txt).
```
Comment (dortegau) [inline README.md]: I think that this link is outdated 😇

Comment (jmonterrubio) [reply README.md]: it is :)

**Pull request merged**